### DISCUSSION TITLE
Table Function

### DISCUSSION THREAD

**Discussion by hmb1** [duckdb/duckdb#2995] [Q&A]
Hi There,
  forgive me for stating the obvious but I not see a  Create Table Function  SQL statement ?
  I see that you can create a Table Function with the DuckDB C++ API  ?

 Thanks ...Henry

**Comment by hannes** [comment]
There is no support for SQL-level table functions yet, we are working on table-level macros as you are of course aware in #2964

**Reply by hmb1** [reply]
OK thanks Hannes

**Comment by hmb1** [comment]
BTW where are the testing Databases/SQL located ?
